charts/perses/values.yaml

```diff
@@ -199,17 +214,3 @@ ingress:
   tls: []
 #    - secretName: ingress-cert
 #      hosts: []
-
-# -- Configure datasources
-# ref: https://github.com/perses/perses/blob/90beed356243208f14cf2249bebb6f6222cb77ae/docs/datasource.md
-datasources:
```

Comment:
This is a breaking change, I don't know how many people is using it today and I'd love to avoid breaking them.

My suggestion is to keep both functionalities and let users decide what they want to use.

Since sidecar is not enabled by default, I don't think it will be an issue.

WDYT?

Reply:
Yes, I think we can consider it, maybe we can mark it as deprecated in this version add a notes do release notes and in some months we can remove it wdyt @ibakshay ?

Reply:
@nicolastakashi, Yes, Thank you very much for the suggestion. 
I have created a PR for the deprecation https://github.com/perses/helm-charts/pull/36:)